### DISCUSSION TITLE
How do I build cattr to test it locally?

### DISCUSSION THREAD

**Discussion by sebastian-an** [cattr-app/server-application#290] [Q&A]
I just want to see how cattr works on my laptop. However I don't know what to put in the manifest when using the docker-compose installation. If I try 'http://localhost' or 'http://localhost:8089' I don't get anything when trying to access the application. It is not clear in the docs where should I go to. On my browser I try to connect to both localhost ports and get nothing. Is there support for this kind of configuration or is it required that I have a server with dns config ready?

**Comment by DanilShapilov** [comment]
Hello. Documentation on docs.cattr.app is outdated.  
Maybe this will help https://github.com/orgs/cattr-app/discussions/163
